locales/es.yaml

```diff
@@ -44,47 +44,47 @@ term:
   internalLevel: Nivel interno
   minInternalLevel: Nivel mínimo interno
   maxInternalLevel: Nivel máximo interno
-  noteDesigner: Notas del Diseñador
-  totalNotes: TOTAL
+  noteDesigner: Diseñador del mapa
+  totalNotes: Total
   region: Región
 
 # Descriptions & Tooltips
 description:
   pending: Pendiente
   loading: Cargando datos…
   newSong: ¡Nueva canción!
-  unlockNeeded: Hay que desbloquear esta canción
+  unlockNeeded: Hay que desbloquear esta canción.
   unavailableInRegion: 'No disponible en {region}'
   copied: ¡Copiado al portapapeles!
-  invalidNoteCounts: Recuentos de NOTA no válidos
-  filterResultEmpty: No se encuentra ninguna hoja que cumpla el filtro.
-  myListEmpty: Haga clic con el botón derecho del ratón o mantenga pulsada una hoja para añadirla a MI LISTA y, a continuación, podrá ver aquí las hojas seleccionadas.
-  drawPoolEmpty: El grupo de dibujo actual está vacío.
+  invalidNoteCounts: Recuentos de nota no válidos
```

Comment:
"NOTE" is uppercased just like "TAP", "HOLD", "SLIDE", "TOUCH", and "BREAK" in maimai, to emphasize that this is a special rhythm game term.